end-to-end-tests/src/test/java/com/amazonaws/blox/CreateDaemonEnvironmentTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox;
+
+import static com.amazonaws.blox.integ.AsynchronousTestSupport.waitOrTimeout;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.groups.Tuple.tuple;
+
+import com.amazonaws.blox.integ.BloxTestStack;
+import com.amazonaws.blox.model.CreateEnvironmentRequest;
+import com.amazonaws.blox.model.DeploymentConfiguration;
+import com.amazonaws.blox.model.DescribeEnvironmentRequest;
+import com.amazonaws.blox.model.DescribeEnvironmentRevisionRequest;
+import com.amazonaws.blox.model.Environment;
+import com.amazonaws.blox.model.EnvironmentRevision;
+import com.amazonaws.blox.model.StartDeploymentRequest;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CreateDaemonEnvironmentTest {
+  private static final String ENVIRONMENT_NAME = "EndToEndTestEnvironment_" + UUID.randomUUID();
```

Comment:
Making this `static final` will reuse the name for all tests, so we have to teardown after each test. If we made it non-static, then each test would get its own environment name, and we'd only have to clean up once after the entire suite has run (but we can revisit this later).

Reply:
y, update environment name to be per test case in new commit.